src/style.css

```diff
@@ -2,2 +1,82 @@
+body {   font-family: 'Roboto', sans-serif;
   margin: 0;
+  background-color: aliceblue;
+  font-size: 12px;
+  text-transform: uppercase;
 }
+.header{
+  background-color: #fff;
+width:  100%;
+height: 60px ;
+display: flex;
+justify-content: space-between;
+
+}
+
+.image{
+  height: 40px;
+  width: 40px;
+  background-color: #fff;
+margin-left: 50px;
+}
+
+.nav{
+  display: inline;
+  text-decoration: none;
+  white-space: nowrap;
+  margin-right: 50px;
+
+
+}
+
+
+.is-active{
+  color:#00ACDC ;
+  position: relative;
+
+  }
+
+  .laptops{
+    color: #00ACDC
+
+  }
+
+a
+{
+  margin:10px;
+  text-decoration: none;
+  color: black;
+  font-weight: bold;
+}
+
+ul
+{list-style-type: none;
+display: flex;
+margin-left: 333px;
+margin-top: 23px;
+
+}
+
+li{
+  justify-content: left;
+}
+
+nav :hover{
```

Comment:
The selector `nav :hover` targets any element that is hovered within a `nav` element. If the intention is to style links on hover, consider using `nav a:hover` instead to specifically target anchor elements.